test/test_tensorexpr.py

```diff
@@ -1505,6 +1505,39 @@ def simple(a, b):
         torch._C._jit_set_te_generate_block_code(val)
         torch._C._jit_texpr_set_fallback_allowed(fall_bk)
 
+    def test_strided_output_preserved(self):
+        def foo(a, b):
+            return a + b - a
```

Comment:
Nit: what's the reason of having two arguments if we're only using one? If having two is critical then it might be a good idea to pass two different inputs in the call site too.